espanol/rezando/paquete/caballero/malvado/malvado.md

```diff
@@ -0,0 +1,5 @@
+
+El caballero negro es malvado. 
+Se dice también que es un hechicero por lo cual es muy poderoso, que incluso podria llegar a derrotar a un ejército entero él solo.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.